src/qibo_client/qibo_client.py

```diff
@@ -255,3 +244,15 @@ def _get_result(self) -> Optional[np.ndarray]:
 
         self.results_path = self.results_folder / "results.npy"
         return qibo.result.load_result(self.results_path)
+
+
+def TII(token: str) -> Client:
```

Comment:
Please move this to another file, e.g. tii.py and import the TII class in the init.